### ISSUE TITLE
Not sure how this works? When I click the button nothing happens.

### ISSUE THREAD

**Issue by azuryst** [deltabravozulu/pimeyesJS#1]
Not sure how this works? When I click the button nothing happens.

**Comment by deltabravozulu** [comment]
Hey mate. Check the[ README.md](https://github.com/deltabravozulu/usefulUserScripts/blob/main/amazonReviewWall) and see if it works like the gif. 

Is your internet connection high speed? Depending on your computer's processing speed and your internet bandwidth, pages with a large amount of image reviews might take a while to load into the script. Maybe test it on a page with like 1 or two image reviews first.

**Comment by deltabravozulu** [comment]
@azuryst any luck?

**Comment by azuryst** [comment]
no, sorry, still does nothing for me. I’ve only tried it in Firefox though.
